controllers/managedocs_controller.go

```diff
@@ -1752,6 +1762,46 @@ func (r *ManagedOCSReconciler) updateMCGCSV(csv *opv1a1.ClusterServiceVersion) e
 	return nil
 }
 
+func (r *ManagedOCSReconciler) reconcileConsolePlugin() error {
+	r.Log.Info("Reconciling ConsolePlugin")
+	if err := r.get(r.ocpConsolePlugin); err != nil && !errors.IsNotFound(err) {
+		return err
+	}
+	if r.ocpConsolePlugin.UID == "" {
+		// ODF Operator has to bring this CR,
+		// so if OCP ConsolePlugin is not ready, we ruturn silently
+		return nil
```

Comment:
Please add a `Log.Info` message specifying that the desired ConsolePlugin wasn't found.

Reply:
Done